.drone.yml

```diff
@@ -45,92 +48,97 @@ pipeline:
       - php occ app:enable files_antivirus
       - php occ app:enable testing
       - php occ app:list
-      - php occ config:system:set trusted_domains 1 --value=server
+      - php occ config:system:set trusted_domains 1 --value=owncloud
       - php occ log:manage --level 0
+    when:
+      matrix:
+        NEED_INSTALL_APP: true
 
-  owncloud-log:
-    image: owncloud/ubuntu:16.04
-    detach: true
-    pull: true
-    commands:
-    - tail -f /var/www/owncloud/data/owncloud.log
-
-  code-compliance-check:
+  configure-app:
     image: owncloudci/php:${PHP_VERSION}
     pull: true
     commands:
-      # currently failing
-      #- make test-php-codecheck
-      - make test-php-lint
-      - make test-php-style
+      - cd /var/www/owncloud/
+      - php occ config:app:set --value "clamav" files_antivirus av_host
+      - php occ config:app:set --value "daemon" files_antivirus av_mode
+      - php occ config:app:set --value "3310" files_antivirus av_port
     when:
       matrix:
-        TEST_SUITE: phpunit
+        TEST_SUITE: api-acceptance
 
-  phpunit:
+  fix-permissions:
     image: owncloudci/php:${PHP_VERSION}
     pull: true
     commands:
-      # dummy clamav required for unit tests
-      - php /var/www/owncloud/apps/files_antivirus/tests/util/avirserver.php&
-      # wait shortly for the server to be available
-      - wait-for-it 127.0.0.1:5555
-      # required to pass the executable tests
-      - ln -s /var/www/owncloud/apps/files_antivirus/tests/util/avir.sh /usr/bin/clamscan
-      - cd /var/www/owncloud/apps/files_antivirus/tests/unit
-      - phpdbg -d memory_limit=4096M -rr ../../../../lib/composer/bin/phpunit --configuration phpunit.xml
+      - chown www-data /var/www/owncloud -R
+      - chmod 777 /var/www/owncloud/tests/acceptance/filesForUpload -R
+      - chmod +x /var/www/owncloud/tests/acceptance/run.sh
     when:
       matrix:
-        TEST_SUITE: phpunit
+        NEED_SERVER: true
 
-  codecov:
-    image: plugins/codecov:2
-    secrets: [codecov_token]
+  owncloud-log:
+    image: owncloud/ubuntu:16.04
+    detach: true
     pull: true
-    files:
-     - 'tests/unit/clover.xml'
+    commands:
+    - tail -f /var/www/owncloud/data/owncloud.log
     when:
-      local: false
       matrix:
-        TEST_SUITE: phpunit
-        PHP_VERSION: 7.0
+        NEED_SERVER: true
 
-  configure-app:
+  owncloud-coding-standard:
     image: owncloudci/php:${PHP_VERSION}
     pull: true
     commands:
-      - cd /var/www/owncloud/
-      - php occ config:app:set --value "clamav" files_antivirus av_host
-      - php occ config:app:set --value "daemon" files_antivirus av_mode
-      - php occ config:app:set --value "3310" files_antivirus av_port
+      - make test-php-style
     when:
       matrix:
-        TEST_SUITE: api-acceptance
+        TEST_SUITE: owncloud-coding-standard
 
-  fix-permissions:
+  phpunit-tests:
     image: owncloudci/php:${PHP_VERSION}
     pull: true
+    environment:
```

Comment:
@patrickjahns yesterday we removed:
```
    - PHP_VERSION=${PHP_VERSION}
```
@individual-it has commented at https://github.com/owncloud/activity/pull/660#discussion_r237371096

--------
it does make a difference if you run drone locally by `drone exec` and specify the PHP version e.g.
`PHP_VERSION=7.2 drone exec` without that line it would not pick the correct version
--------

thoughts?

Reply:
Seems to be strange, as the image `image: owncloudci/php:${PHP_VERSION}` specifies the container to use. 

The `environment` key is only to inject the variable in the environment inside the container. Where does that environment need to be set inside the container?

Reply:
@individual-it your input is needed - please explain

Reply:
I was wrong, its not needed here.
I had problems running core acceptance tests
running `PHP_VERSION=7.1 TEST_SUITE=api BEHAT_SUITE=apiCapabilities DB_TYPE=mariadb USE_SERVER=true INSTALL_SERVER=true CHOWN_SERVER=true drone exec` in core
gives me:
`Running all API tests tagged ~@skipWhenTestingRemoteSystems&&~@skipOnOcV11&&~@skipOnOcV11.0&&~@skipOnOcV11.0.0&&@api&&~@skip`
because there is no `-BEHAT_SUITE=${BEHAT_SUITE}` set in the `.drone.yml` in core the setting is not picked up in a manual run

sorry for the noice